core/src/main/scala/kafka/coordinator/GroupCoordinator.scala

```diff
@@ -401,6 +407,31 @@ class GroupCoordinator(val brokerId: Int,
     }
   }
 
+  def handleListGroups(): (Errors, List[GroupOverview]) = {
```

Comment:
That's an interesting thought. Do you want to add that case here or after we merge KAFKA-2017?